_posts/how_API_works.md

```diff
@@ -0,0 +1,714 @@
+The Blogger API enables you to integrate Blogger content with your application by using the REST APIs. Before you begin, you will need to set up authorization.
+
+Introduction:
```

Comment:
Please edit the headings properly acc to markdown editing style like here Introduction should be in h2 / h3